src/environments/environment.ts

```diff
@@ -0,0 +1,17 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+  production: false
+};
+export const BASE_URL = 'https://blue-bottle-api-test.herokuapp.com';
```

Comment:
Good idea extracting the base url in an environment file.

Reply:
Yes, but I spent 4 hours figuring out that I need to pass a headers object, accepting application/json otherwise the call would fail, and left it in the service as a plain string. Forgot to replace with the BASE_URL when I was finished.